qualtran/bloqs/basic_gates/swap.py

```diff
@@ -103,6 +115,21 @@ def on_classical_vals(
     def adjoint(self) -> 'Bloq':
         return self
 
+    def get_ctrl_system(self, ctrl_spec: 'CtrlSpec') -> tuple['Bloq', 'AddControlledT']:
+        if ctrl_spec != CtrlSpec():
+            return super().get_ctrl_system(ctrl_spec=ctrl_spec)
+
+        cbloq = TwoBitCSwap()
```

Comment:
please reserve this variable name for composite bloqs (sorry). `ctrl_bloq` if you need it to be agnostic or (preferred) `cswap`

Reply:
renamed to `cswap`